services/web-app/components/page-tabs/page-tabs.module.scss

```
@@ -0,0 +1,45 @@
+//
+// Copyright IBM Corp. 2022, 2022
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+.container {
+  @include breakpoint(md) {
+    margin-left: -$spacing-05;
+  }
+
+  &::before {
+    position: absolute;
+    z-index: 0;
+    left: 0;
+    width: 100vw;
+    height: $spacing-09;
```

```suggestion
    height: $spacing-08;
```

to match default tab height